tests/cases/fourslash/navigationBarAnonymousClassAndFunctionExpressions.ts

```diff
@@ -62,7 +62,7 @@ verify.navigationTree({
                     "childItems": [
                         {
                             "text": "foo",
-                            "kind": "function"
+                            "kind": "property"

```

Comment:
I'm a little surprised this changed.  You might want to ask someone with more JS experience than me whether they have a preference.

Reply:
It should be `method` now (which appears to have the same icon as `function`).